src/main/java/Duke.java

```diff
@@ -1,10 +1,144 @@
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    private static ArrayList<Task> tasks = new ArrayList<Task>();
+    private static final String LINE_BREAK = "______________________________________________\n";
+
+    public static void printList() { //prints list of tasks
+        System.out.println(LINE_BREAK);
+        for (int i = 0; i < tasks.size(); i++) {
+            System.out.println( Integer.toString(i + 1) + "." + tasks.get(i).toString());
+        }
+        System.out.println(LINE_BREAK);
+    }
+
+    public static void printMarkedTask(Task markedTask) { //prints that the task is marked
+        System.out.println(LINE_BREAK + "Nice! I've marked this task as done: ");
+        System.out.println("  " + markedTask.toString());
+        System.out.println();
+    }
+
+    public static void printUnmarkedTask(Task unmarkedTask) { //prints that the task is not done yet
+        System.out.println(LINE_BREAK + "OK, I've marked this task as not done yet: ");
+        System.out.println("  " + unmarkedTask.toString());
+        System.out.println();
+    }
+
+    public static void printAddedTask(Task task) { //prints that the task has been added to list
+        System.out.println(LINE_BREAK + "Got it. I've added this task: ");
+        System.out.println(" " + task.toString());
+        System.out.println("Now you have " + Integer.toString(tasks.size()) + " tasks in the list \n" + LINE_BREAK);
+    }
+
+    public static ToDo getToDo(String[] inputWords) { //returns a todo object from the user input sentence
+        String[] taskNameWords = Arrays.copyOfRange(inputWords, 1, inputWords.length);
+        String taskName = String.join(" ", taskNameWords);
+        ToDo toDo = new ToDo(taskName);
+        return toDo;
+    }
+
+    public static Deadline getDeadline(String[] inputWords) { //returns a Deadline object from the user input sentence
+        int indexOfBy = Arrays.asList(inputWords).indexOf("/by");
```

Comment:
Avoid magic strings. Consider putting the strings into a descriptive variable. For example, `/by` can be put into the variable `DEADLINE_DATA_PREFIX_BY`.